g2p/mappings/langs/generated/crk-ipa_to_eng-ipa.json

```diff
@@ -13,19 +7,19 @@
     },
     {
         "in": "i\u02d0",
-        "out": "i",
+        "out": "e\u02d0",
         "context_before": "",
         "context_after": ""
     },
     {
-        "in": "o\u02d0",
-        "out": "o\u02d0",
```

Comment:
Ideally, the `crk-ipa` -> `eng-ipa` would know to convert **oː** to **oʊ**, but making this map straight from SRO to **oʊ** works too!